Makefile

```diff
@@ -15,7 +15,9 @@ GIT_REPO=metal-go
 PACKAGE_PREFIX=metal
 PACKAGE_MAJOR=v1
 
-SWAGGER=docker run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local ${IMAGE}
+# Pull in custom generator jar so I can use unmerged bugfixes
+OPENAPI_GENERATOR=/Users/ctreatman/Documents/code/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar
```

Comment:
For faster turnaround on identifying openapi-generator issues & testing fixes, I updated the `Makefile` on this branch to use a locally-built openapi-generator.

The path is hard-coded with my local username, so if we find that this custom generator approach has to persist, we will need to either make the path more flexible or require that openapi-generator be checked out as a submodule.

The openapi-generator fork I am using is [ctreatma/openapi-generator](https://github.com/ctreatma/openapi-generator).